src/Tasks/XslTransformation.cs

```diff
@@ -467,6 +467,8 @@ public XslCompiledTransform LoadXslt(bool useTrustedSettings)
                         Type t = FindType(assemblyPath, typeName);
                         xslct.Load(t);
                         break;
+#else
+                        throw new Exception("This is not supported on .NET Core.");
```

Comment:
```suggestion
                        throw new PlatformNotSupportedException("Precompiled XSLTs not supported on .NET Core.");
```

Reply:
And can you catch this specifically in the relevant block in `Execute` and log a new MSB error?

Reply:
Yes. Throwing blanket Exception makes me nervous.

Reply:
Got it